### DISCUSSION TITLE
It seems that the editor can't be opened. why?

### DISCUSSION THREAD

**Discussion by sdy1978** [AykutSarac/jsoncrack.com#47] [Q&A] [answered]
It seems that the editor can't be opened. why?
below is the picture, it always shows this screen.
![image](https://user-images.githubusercontent.com/105090833/171113125-a512121d-3063-4ead-a14a-792264336a16.png)

**Comment by AykutSarac** [comment, marked as answer]
Hello,

Just pushed a fix, let me know if it will work for you.

**Reply by AykutSarac** [reply]
I'll consider the issue has been fixed as per the increase in visitors activity coming from China.
